akka-docs/rst/scala/mailboxes.rst

```diff
@@ -153,54 +153,57 @@ Akka comes shipped with a number of mailbox implementations:
   - Bounded: No
 
   - Configuration name: "akka.dispatch.UnboundedControlAwareMailbox"
+  
+* **UnboundedPriorityMailbox**
 
-Previously available mailbox implementations which were deprecated because they might block:
-
-* **BoundedMailbox**
-
-  - Backed by a ``java.util.concurrent.LinkedBlockingQueue``
+  - Backed by a ``java.util.concurrent.PriorityBlockingQueue``
 
-  - Blocking: Yes
+  - Delivery order for messages of equal priority is undefined - contrast with the UnboundedStablePriorityMailbox
 
-  - Bounded: Yes
+  - Blocking: No
 
-  - Configuration name: "bounded" or "akka.dispatch.BoundedMailbox"
+  - Bounded: No
 
-* **UnboundedPriorityMailbox**
+  - Configuration name: "akka.dispatch.UnboundedPriorityMailbox"
+  
+* **UnboundedStablePriorityMailbox**
 
-  - Backed by a ``java.util.concurrent.PriorityBlockingQueue``
+  - Backed by a ``java.util.concurrent.PriorityBlockingQueue`` wrapped in an ``akka.util.PriorityQueueStabilizer``
 
-  - Delivery order for messages of equal priority is undefined - contrast with the UnboundedStablePriorityMailbox
+  - FIFO order is preserved for messages of equal priority - contrast with the UnboundedPriorityMailbox
 
-  - Blocking: Yes
+  - Blocking: No
 
   - Bounded: No
 
-  - Configuration name: "akka.dispatch.UnboundedPriorityMailbox"
+  - Configuration name: "akka.dispatch.UnboundedStablePriorityMailbox"
 
-* **BoundedPriorityMailbox**
+Other bounded mailbox implementations which will block the sender if the capacity is reached and
+configured with non-zero ``mailbox-push-timeout-time``. 
 
-  - Backed by a ``java.util.PriorityQueue`` wrapped in an ``akka.util.BoundedBlockingQueue``
+.. note:: The following mailboxes should only be used with zero ``mailbox-push-timeout-time``.
```

Comment:
It's probably difficult to see what I changed. I have moved the UnboundedPriorityMailbox  UnboundedStablePriorityMailbox up to the first list of mailboxes, and then comes this description about potentially blocking mailboxes and the list of such bounded mailboxes.